plugins/locator/stdloc/descriptions/global_stdloc.xml

```diff
@@ -17,7 +17,7 @@
 				<group name="profile">
 					<struct type="stdloc profile" link = "StdLoc.profiles">
 
-						<parameter name="method" type="string" default="GridSearch+LeastSquares">
+						<parameter name="method" type="string" default="LeastSquares" values="LeastSquares,GridSearch,OctTree,GridSearch+LeastSquares,OctTree+LeastSquares" >
 							<description>The location method to use: LeastSquares, GridSearch, OctTree,
 							 GridSearch+LeastSquares or OctTree+LeastSquares.
```

Comment:
The list of methods can be dropped here, since the  tag value is used.

Reply:
I don't know, because the list of possible values is not visible on scconfig. So a user would not know the available options

Reply:
With SeisComP in version nightly of upcoming 7, this list is visible in scconfig and in the documentation. Example: scmag with magnitudes.average . We have plans to evaluate this tag in scconfig providing a selection box, etc.

You may keep it for now but later it could be removed.

Reply:
Interesting, I didn't know the feature was already available. So you can remove it from the master branch and not merge the commit to v6 branch.

Reply:
yes, you can. But the value and range tags will simply be ignored for SC v. 6.4. There is no problem with adding it to master.